### PR TITLE
Add a note about webpack into the README

### PR DESCRIPTION
**Pull request by VarLog**
This note will be helpful for peoples who want to use pica into their applications bundled with webpack and who run into troubles that described into https://github.com/nodeca/pica/issues/149, https://github.com/nodeca/pica/issues/70, https://github.com/nodeca/pica/pull/72.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,32 @@ load module in node.js style as `require('pica')` - your project MUST
 be compiled with [browserify](https://github.com/substack/node-browserify)
 to properly use Web Workers. In other case - use `require('pica/dist/pica')`.
 
+webpack:
+
+If you use Webpack to bundle your application, you must to define a resolve
```
Comment (puzrin) [inline README.md]: `must` is a bit too "strong". This looks like official and the only possible correct use.

Could you rephrase this, to make it look llike "notice" or "advice"?

File: README.md [hunk 2]
```diff
@@ -80,6 +80,32 @@ load module in node.js style as `require('pica')` - your project MUST
 be compiled with [browserify](https://github.com/substack/node-browserify)
 to properly use Web Workers. In other case - use `require('pica/dist/pica')`.
 
+webpack:
```
Comment (puzrin) [inline README.md]: Probably, will be better as `**Webpack notice**`

File: README.md [hunk 3]
```diff
@@ -80,6 +80,32 @@ load module in node.js style as `require('pica')` - your project MUST
...
+
+If you use Webpack to bundle your application, you must to define a resolve
+alias into your webpack config, like this:
+
+```js
+{
+    ....
```
Comment (puzrin) [inline README.md]: 1. I think it's safe to drop `...` everywhere and cut down total height. It's important to keep readme compact.
2. Also, please, use 2 spaces for indent, not 4.

**Comment by puzrin** [comment]
LGTM except some minor notes.

PS. If you are busy - let me know, i'll merge as is and will fix the rest myself.

**Comment by VarLog** [comment]
OK, I fixed issues

**Comment by puzrin** [comment]
Thanks! If you ever find a way to make pica work without hacks - let me know. I'm not strong in browser-related things.
